### PR TITLE
New Cashjs Cheat Sheet

### PR DESCRIPTION
**Pull request by bernardodsanderson**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ X] New Instant Answer
    - [X ] Cheat Sheets: **`New Cashjs Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

A new cheat sheet for Cashjs (or simply Cash), a absurdly small JavaScript library similar to jQuery.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/cashjs_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @bernardodsanderson

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Cash Cheat Sheet](https://duck.co/ia/view/cash_cheat_sheet)

**Description**: A cheat sheet for Cash JavaScript library, an absurdly small jQuery alternative.

**Example Query**: [cash cheat sheet](https://beta.duckduckgo.com/?q=cash cheat sheet)

**Tab Name**: Cheat Sheet

**Source**: https://github.com/kenwheeler/cash

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/cash_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/cash_cheat_sheet)*

**Comment by daxtheduck** [comment]
@bernardodsanderson Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cash cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cash_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cash cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cash_cheat_sheet_mobile.png)

**Comment by mintsoft** [comment]
@bernardodsanderson Apologies I've repushed the build and the tests have passed now; this should be auto deployed to beta soon!

File: share/goodie/cheat_sheets/json/cashjs.json [hunk 1]
```diff
@@ -0,0 +1,330 @@
+{
+    "id": "cashjs_cheat_sheet",
+    "name": "Cashjs",
+    "description": "A cheat sheet for Cash JavaScript library, an absurdly small jQuery alternative.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/cashjs.json]: Lets remove the period at  end of desc

File: share/goodie/cheat_sheets/json/cashjs.json [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+                "val": "Check if the argument is a function."
+            },
+            {
+                "key": "$.isNumeric()",
+                "val": "Check if the argument is numeric."
+            },
+            {
+                "key": "$.isString()",
+                "val": "Check if the argument is a string."
+            }
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/cashjs.json]: looks better if periods at end of val fields are also removed

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cash cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cash_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/cashjs.json [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+    "name": "Cashjs",
+    "description": "A cheat sheet for Cash JavaScript library, an absurdly small jQuery alternative",
+
+    "metadata": {
+        "sourceName": "Github",
+        "sourceUrl" : "https://github.com/kenwheeler/cash"
+    },
+
+    "aliases": [
+        "cash"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/cashjs.json]: Not sure about this - how about `cash js`?

File: share/goodie/cheat_sheets/json/cashjs.json [hunk 4]
```diff
@@ -0,0 +1,330 @@
+{
+    "id": "cashjs_cheat_sheet",
+    "name": "Cashjs",
+    "description": "A cheat sheet for Cash JavaScript library, an absurdly small jQuery alternative",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/cashjs.json]: We could just use `The Cash JavaScript .....`

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=cash cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/cash_cheat_sheet_mobile.png?skS6xPMwSn)

**Comment by GuiltyDolphin** [comment]
@moollaza Could we get the IA page updated to `cashjs_cheat_sheet`.

**Comment by bernardodsanderson** [comment]
Let me know if you would like anything else changed, I've made all the changes you asked so far.

**Comment by jophab** [comment]
@bernardodsanderson I think this can be merged now.

**Comment by bernardodsanderson** [comment]
Sorry @jophab, how do I go about doing that? I didn't see how on the site, maybe I make a new page?

**Comment by jophab** [comment]
@bernardodsanderson Dont worry @moollaza will do it.

**Comment by alohaas** [comment]
Hey @jophab @bernardodsanderson @GuiltyDolphin, @moollaza is away at the moment, but I will make the IA Page change.

**Comment by moollaza** [comment]
I've updated the IA Page link above and restarted the test to confirm everything is good.

After that it's good to merge

**Comment by daxtheduck** [comment]
@bernardodsanderson this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**